data/Templates/Ccda/Resource/_Patient.liquid

```diff
@@ -57,11 +57,13 @@
                         {
                             "system":"urn:ietf:bcp:47",
                             "code":"{{ languageCommunication.languageCode.code }}",
-                            "display":"{{ languageCommunication.languageCode.code | get_property: 'ValueSet/Language' }}",
+                            "display":"{{ languageCommunication.languageCode.code | downcase | get_property: 'ValueSet/Language' }}",
```

Comment:
If the input parameter is null or empty, or even not string type, will `downcase` handle it appropriately?

Reply:
I double-check and test it, it works correctly, no result will be generated for null or empty result, the original value will be converted to string for int, decimal or boolean type

Reply:
Got it, thanks!